Assignment_2-data-emma-luk.js

```diff
@@ -0,0 +1,101 @@
+﻿// Emma Luk
+// Assignments 2 Using the revealing module pattern, build a Date utility module 
+
+var myDate = (function iife() {
+    var displaysModuleDate = null;
+
+    //private function: date instantiation
+    function _initDate() {
+        return new Date();
+    }
+
+    //check here that date is set: if not, initialize date to now
+    function setDate(date) {
+        if (date === undefined && null) {
+            displaysModuleDate = _initDate();
+        }
+        else if (typeof date === 'number' && !isNaN(date)) {
+            displaysModuleDate = new Date(date);
+        }
+        else if (typeof date === 'object') {
+            displaysModuleDate = date;
+        }
+        else {
+            console.log("sorry!! wrong type");
+        }
+    }
+
+    function getDate(objFormat) {
+        if (displaysModuleDate === null && undefined) {

```

Comment:
The same comment applies here that was made above. Check each value individually, and use the || operator instead: `if (displayModuleDate === null || displayModuleDate === undefined)` Refer to the comment about the similar check you make above for more details.